v0/destinations/snapchat_conversion/transform.js

```diff
@@ -0,0 +1,385 @@
+const get = require("get-value");
+const { logger } = require("handlebars");
+const { EventType } = require("../../../constants");
+
+const {
+  defaultPostRequestConfig,
+  constructPayload,
+  defaultRequestConfig,
+  removeUndefinedAndNullValues,
+  getFieldValueFromMessage,
+  defaultBatchRequestConfig,
+  getErrorRespEvents,
+  getSuccessRespEvents
+} = require("../../util");
+const ErrorBuilder = require("../../util/error");
+const {
+  ENDPOINT,
+  eventNameMapping,
+  mappingConfig,
+  ConfigCategory,
+  MAX_BATCH_SIZE
+} = require("./config");
+const {
+  msUnixTimestamp,
+  getHashedValue,
+  getItemIds,
+  getPriceSum,
+  getDataUseValue,
+  getNormalizedPhoneNumber
+} = require("./util");
+
+function trackResponseBuilder(message, { Config }) {
+  let event = get(message, "event");
+  if (!event) {
+    throw new ErrorBuilder()
+      .setMessage("[Snapchat] :: Event name is required")
+      .setStatus(400)
+      .build();
+  }
+  event = event.trim().replace(/\s+/g, "_");
+
+  const { apiKey,pixelId, snapAppId, appId } = Config;
+  const eventConversionType = message?.context?.snapchat_conversion_type?.toUpperCase();
```

Comment:
Make sure there is no parsing error for this kind of implementation. In my local it shows as parsing error.

Reply:
Ya, It's working fine.